### ISSUE TITLE
Can only see part of Session Sync if it's in the overflow menu

### ISSUE THREAD

**Issue by bottleblue** [ReDEnergy/SessionSync#109]
I can't see the entire window if it's opened from the overflow menu.

[](https://imgur.com/tmQ8IO3)

**Comment by ReDEnergy** [comment]
Yeah... well UI was never intended to be made useful like that :D
![image](https://user-images.githubusercontent.com/1395510/118860460-176ce980-b8e4-11eb-889d-cca03d26e15e.png)

**Comment by ReDEnergy** [comment]
Oh. My bad. Sorry but there is no support for the DropMenu UI. Firefox enforces that resolution, you can't make it larger. Not something I can fix sadly. The only usable way is to place the SessionSync UI in the Addon Bar next to all others addon icons.

**Comment by bottleblue** [comment]
I didn't notice the resize handles. But even dragging the middle handle to the limit is not wide enough. I still can't see the hamburger menu on the right and only some of the mouseover text.
![](https://i.imgur.com/ya9G9dc.jpg)

If you can't make the window larger is it possible to make the icons and mouseover text a little smaller and also move the hamburger menu next to the search box? The two timestamps also seem redundant? 
![](https://i.imgur.com/zHdverJ.jpg)


When you're mousing over the website names in the right hand window maybe make the url a popup text instead of next to search box.
![](https://i.imgur.com/VpbHj0K.jpg)

**Comment by ReDEnergy** [comment]
Sorry, but making Session Sync UI to be useful inside that small area is a really complicated and since I have a lot more important features to add it certainly not on any priority list. I hate the fact that Firefox blocks us to even use more than 800 x 600 resolution. It's stupid we are in 2021 and we can't even make an addon popup larger than 800 x 600 (welcome to the future). I tried to make the UI useful inside that max resolution, and maybe leave some space for adjustment for each individual to change as he likes but can't really put that much effort to make something in a 400px wide panel, it's simply stupid for this kind of addon.

The wait it should look and how I designed it.

![image](https://user-images.githubusercontent.com/1395510/118880625-bd2b5300-b8fa-11eb-95da-a46505ef66b9.png)

**Comment by bottleblue** [comment]
I see. No problem it's not that big of a deal. Just thought I'd ask.  Thx for replying.

**Comment by bottleblue** [comment]
I was trying out another session manager addon and it has the same problem too in the overflow menu but it solved it by adding a horizontal scrollbar at the bottom. Maybe you could do that also?

![1](https://user-images.githubusercontent.com/3678136/131395509-0d05dcd1-bb69-4a1e-93e9-53110895df4e.jpg)
![2](https://user-images.githubusercontent.com/3678136/131395556-80ccd81d-d102-4ecc-8a81-ce197326378d.jpg)
